engineering/python-style-guide.md

```diff
@@ -0,0 +1,13 @@
+source{d} Python Style Guide
+
+1. Contributing to existing projects - do not change the style.
+2. [PEP8](https://www.python.org/dev/peps/pep-0008/) with 99 char line length limit
+3. Indentation: 4 whitespaces
```

Comment:
This is already defined in PEP8: https://www.python.org/dev/peps/pep-0008/#indentation
**Use 4 spaces per indentation level.**